Imagine/Cache/Resolver/FlysystemResolver.php

```diff
@@ -132,7 +141,7 @@ public function store(BinaryInterface $binary, $path, $filter)
         $this->flysystem->put(
             $this->getFilePath($path, $filter),
             $binary->getContent(),
-            ['visibility' => $this->visibility]
```

Comment:
Yeah, the the travis file always excluded `flysystem` from PHP 5.3 runs. I've been using the assumption FlySystem requires a higher PHP version, but if that's not the case we should obviously be testing it! I'll take a look at flysystems composer when I get a second.